mrp_bom_attribute_match_duplicate_kit/tests/common.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form, SavepointCase
+
+
+class TestProductTemplateKitCommon(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestProductTemplateKitCommon, cls).setUpClass()
+        # Attribute Color
+        cls.attribute_color = cls.env.ref(
+            "product.product_attribute_2", raise_if_not_found=False
+        )
+
+        # Attribute Color Values
+        cls.attribute_color_white = cls.env.ref(
+            "product.product_attribute_value_3", raise_if_not_found=False
+        )
+        cls.attribute_color_black = cls.env.ref(
+            "product.product_attribute_value_4", raise_if_not_found=False
+        )
+
+        # Attribute Legs
+        cls.attribute_legs = cls.env.ref(
+            "product.product_attribute_1", raise_if_not_found=False
+        )
+
+        # Attribute Legs Values
+        cls.attribute_legs_steel = cls.env.ref(
+            "product.product_attribute_value_1", raise_if_not_found=False
+        )
+        cls.attribute_legs_aluminium = cls.env.ref(
+            "product.product_attribute_value_2", raise_if_not_found=False
+        )
```

Comment:
**quibble(non-blocking)**: I don't think we should have `raise_if_not_found=False` here because if they are not found, you'll have `False` and then the tests would (or should) fail, and it'll be harder to troubleshoot why. If we simply have the error raised immediately when doing `cls.env.ref()` it's easier to realize that maybe the XML ID of an attribute changed.

Reply:
@aleuffre  Agree with you!